app/assets/src/components/views/discovery/mapping/BaseMap.jsx

```diff
@@ -27,10 +27,11 @@ class BaseMap extends React.Component {
   };
 
   render() {
-    const { mapTilerKey, hoverTooltip, markers, popups } = this.props;
+    const { mapTilerKey, tooltip, markers, popups } = this.props;
     const { viewport } = this.state;
 
-    const styleID = "6041e288-ea0e-4a64-b0f6-78172d56a657";
+    // MapTiler map name: "2019-04-24"
+    const styleID = "f0e7922a-43cf-4dc5-b598-17ae1f56d2f4";
```

Comment:
Nit: can we move this to the top as a constant? Maybe "IDSEQ_MAP_STYLE_ID".

Reply:
👍 done